python/pyspark/sql/functions.py

```diff
@@ -153,7 +173,7 @@ def _():
 # math functions that take two arguments as input
 _binary_mathfunctions = {
     'atan2': 'Returns the angle theta from the conversion of rectangular coordinates (x, y) to' +
-             'polar coordinates (r, theta).',
+             'polar coordinates (r, theta). Units in radians.',
```

Comment:
Most libraries seem to default to radians. However, I checked the R, numpy, and MATLAB notes for common trigonometry functions, and they all note the units in the function documentation, e.g.:
https://docs.scipy.org/doc/numpy/reference/generated/numpy.sin.html
https://www.mathworks.com/help/matlab/ref/sin.html